src/Commands/Site/CreateCommand.php

```diff
@@ -749,6 +762,37 @@ protected function createExternallyHostedSite($site_name, $label, Upstream $upst
         $this->log()->notice('Site "{site}" created successfully with GitHub repository!', ['site' => $site->getName()]);
         $this->log()->notice('GitHub Repository: {url}', ['url' => $target_repo_url]);
         $this->log()->notice('Pantheon Dashboard: {url}', ['url' => $site->dashboardUrl()]);
+        if ($repo_cloned) {
+            $this->log()->notice('Code repository cloned successfully to the current directory.');
+        }
+    }
+
+    // Clone the repository using the converted SSH URL.
+    private function cloneRepo($repo_url)
+    {
+        $repo_url = $this->convertToSsh($repo_url);
+
+        // Run git clone command
+        $process = new Process(['git', 'clone', $repo_url]);
+        $process->run();
+
+        if (!$process->isSuccessful()) {
+            throw new ProcessFailedException($process);
+        }
+
+        echo "Repo cloned to current directory.\n";
+        echo $process->getOutput();
```

Comment:
I think we should use `$this->log()->notice` to print messages instead of echo

Reply:
Done. Thanks